data/actionsmap/yunohost.yml

```diff
@@ -1289,6 +1289,36 @@ tools:
 
 
 #############################
+#         Migrations        #
+#############################
+migrations:
```

Comment:
Debugging and be able to go backward/forward in the migrations if needed if everything has crashed.